easybuild/easyconfigs/g/GEOS/GEOS-3.6.2-intel-2018.01-Python-3.6.3.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.6.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'intel', 'version': '2018.01'}
+
+source_urls = ['http://download.osgeo.org/geos/']
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['GEOS-%(version)s_fix-Python3.patch']
+checksums = [
+    '045a13df84d605a866602f6020fc6cbf8bf4c42fb50de237a08926e1d7d7652a'  # geos-3.6.2.tar.bz2
```

Comment:
@reedts You're missing a comma (`,`) here after `...a'`.

Pro tip: use `eb --inject-checksums GEOS-3.6.2-intel-2018.01-Python-3.6.3.eb`

Reply:
@boegel Thanks, works like charm, updating it now.